docs/versionhistory.rst

```diff
@@ -5,12 +5,12 @@ This library adheres to `Semantic Versioning 2.0 <https://semver.org/#semantic-v
 
 **UNRELEASED**
 
-- Fixed ``warn_on_error()`` not showing where the type violation actually occurred
-- Fixed local assignment to ``*args`` or ``**kwargs`` being type checked incorrectly
-- Fixed ``TypeError`` on ``check_type(..., None)``
 - **BACKWARD INCOMPATBILE** Made ``check_type`` independent of ``typeguard.config``, it
```

Comment:
Fixed a typo and added () to indicate function.
```suggestion
- **BACKWARD INCOMPATIBLE** Made ``check_type()`` independent of ``typeguard.config``, it
```

Reply:
This exceeds line-length, I'll push a re-formatting :)

Reply:
Oh, noticed rest of file doesn't adhere to line-length limit, so I'll leave it :)